source/crud/tests/unified/bulkWrite-arrayFilters-clientError.yml

```diff
@@ -0,0 +1,101 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
+
+description: bulkWrite-arrayFilters-clientError
+schemaVersion: '1.1'
+createEntities:
+  -
+    client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      uriOptions:
+        retryReads: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: client0
+      databaseName: &database_name crud-v2
+  -
+    collection:
+      id: &collection0 collection0
+      database: database0
+      collectionName: &collection_name crud-v2
+runOnRequirements:
+  -
+    maxServerVersion: 3.5.5
+initialData:
+  -
+    collectionName: *collection_name
+    databaseName: *database_name
+    documents:
+      -
+        _id: 1
+        'y':
+          -
+            b: 3
+          -
+            b: 1
+      -
+        _id: 2
+        'y':
+          -
+            b: 0
+          -
+            b: 1
+tests:
+  -
+    description: 'BulkWrite on server that doesn''t support arrayFilters'
```

Comment:
This is odd, as the original `description` used double quotes around the string to avoid a conflict with the single qutoe within. Is the sequence of two single quotes an escaping mechanism in YAML?

Reply:
Yes, that's how it's escaped. I couldn't find a way to change what quotes are used when dumping YAML, which is why everything is quoted using single quotes now.